pandas/core/window.py

```diff
@@ -503,6 +503,7 @@ class Window(_Window):
     * ``general_gaussian`` (needs power, width)
     * ``slepian`` (needs width).
 
+    If ``win_type=None`` a boxcar window is used.
```

Comment:
If we add this, I think we should explain what a boxcar window is (as now in the default case the user does not need to worry about it, and those assuming you just get a uniform hard-cut window are correct)

Reply:
Come to think of it, none of the other windows are explained either! Are the windows used custom pandas ones, or are they just the normal numpy ones? And if they're pandas ones do they have their own bit of documentation to link to?

Reply:
these are all scipy windows
certainly take links to that documentation (in doc string and in computation.rst)